src/test/java/duke/storage/StorageTest.java

```diff
@@ -0,0 +1,49 @@
+package duke.storage;
+
+import duke.task.TaskList;
+import duke.task.Todo;
+import org.junit.jupiter.api.Test;
```

Comment:
Similar to the previous suggestion, perhaps import `org.junit.jupiter.api.Test` should be separated from the `duke` import statements?